bins/packages/storcli/storcli.sh

```diff
@@ -1,17 +1,18 @@
+STORCLI_VERSION="1.21.06"
+
 dependencies_storcli() {
     apt-get update
-    apt-get install wget
 
     echo "[+] updating soruces list"
     wget -O - http://archive.thomas-krenn.com/tk-archive.gpg.pub | apt-key add -
-    cd /etc/apt/sources.list.d
-    wget http://archive.thomas-krenn.com/tk-main-xenial.list
-    wget http://archive.thomas-krenn.com/tk-optional-xenial.list
+
+    wget -O /etc/apt/sources.list.d/tk-main-xenial.list http://archive.thomas-krenn.com/tk-main-xenial.list
+    wget -O /etc/apt/sources.list.d/tk-optional-xenial.list http://archive.thomas-krenn.com/tk-optional-xenial.list
     cd
```

Comment:
You don't change directory before, this could even do weird stuff with the `pushd` from `build_storcli`